README.md

```diff
@@ -104,17 +104,30 @@ But it's best to use ghc 7.8.x, where x >= 4. ghc 7.10 is not yet
 supported. If your package manager doesn't have a suitable ghc version, you
 can download it manually from https://www.haskell.org/ghc/.
 
-Clone the Thentos repository from GitHub. Afterwards create a cabal sandbox
-in the same directory and install Thentos:
+Clone the Thentos repository from GitHub. There are several packages in
+this repository:
+
+* `thentos-core`: the core package
+* `thentos-tests`: tests for `thentos-core`
+* `thentos-adhocracy`: integration with the Adhocracy software
+* ...
+
+You'll need to build `thentos-core` in any case. `thentos-tests` is only
+required if you want to run tests. Whether you need other packages will
+depend on your use case.
+
+We recommend building all required package into the same sandbox. To
+Install `thentos-core`, change into the `thentos-core` directory and
+execute the following commands:
 
 ```shell
-$ cabal sandbox init
-$ cabal install --enable-tests --enable-documentation --dependencies-only
+$ cabal sandbox init --sandbox=../.cabal-sandbox
```

Comment:
We should probably mention that you need to `cabal sandbox init` in the repo root first.